actions/telegram-send/telegram-send.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Exo\Toolkit\Runner;
+use Telegram\Bot\Api;
+
+require_once __DIR__.'/../../autoload.php';
+
+$run = Runner::run(function ($request) {
+    $input = $request['input'];
+
+    $token = $input['token'];
+    $channel = $input['channel'];
+    $text = $input['text'];
+
+    $telegram = new Api($token);
+
+    $telegram->sendMessage(
+        [
+            'chat_id' => $channel ,
+            'parse_mode' => 'markdown',
+            'text' => 'Hi @BotFather  send message from api',
```

Comment:
Bug: this text is hardcoded, and should be variable `$text`. Additionally there is a redundant space, and API (abbreviation) should be in all capitals